decompiler/config/jak1_ntsc_black_label/inputs.jsonc

```diff
@@ -262,5 +262,32 @@
   //"audio_dir_file_name": "jak1/VAG",
   "audio_dir_file_name": "",
 
-  "streamed_audio_file_names": ["VAGWAD.ENG", "VAGWAD.JAP"]
+  "streamed_audio_file_names": ["VAGWAD.ENG", "VAGWAD.JAP"],
+  
+  "levels_to_extract":[
```

Comment:
good idea to move this.  maybe we can eventually have it be automatic based on DGO name.